Openshift4/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+ 
+When contributing to this repository, please first discuss the change you wish to make via slack, issue, email, or any other method with the owners of this repository before making a change.
```

Comment:
since this is becoming public. is this comment for external developers? In that case we should not mention slack but write actually email they should contact at.

Reply:
This contributors file is the template we always use.